src/util/charmrun-src/charmrun.C

```diff
@@ -4445,27 +4446,38 @@ static void envCat(char *dest, LPTSTR oldEnv)
 
 /* simple version of charmrun that avoids the sshd or charmd,   */
 /* it spawn the node program just on local machine using exec. */
-static void start_nodes_local(std::vector<nodetab_process> & process_table)
+struct local_nodestart
```

Comment:
maybe change to class

Reply:
If everything inside is public I prefer using struct.

Reply:
cmdLine doesn't even need to be public.